src/test/java/com/training/services/tests/VoyagesTest.java

```diff
@@ -55,16 +57,18 @@ public void testVoyageCurrentSailDateValidate() {
     @Test
     public void testVoyageSailDurationValidate() {
         Response voyageResponse = getResponse();
-        logger.info("status->" + voyageResponse.getStatus());
-        logger.info("totalVoyagesCount->" + voyageResponse.getPayload().getVoyages().size());
+
         Assertions.assertThat(voyageResponse.getStatus()).isEqualTo("200")
                 .as(" status is not 200");
+
         List<Voyages> voyages = voyageResponse.getPayload().getVoyages();
-        for (int i = 0; i < voyages.size(); i++) {
-            int duration = Integer.parseInt(voyages.get(i).getDuration());
-            softAssert.assertThat(duration).isLessThan(10).as(voyages.get(i).getDuration() +
+
+        IntStream.range(0, voyages.size()).forEach(index -> {
+            int duration = Integer.parseInt(voyages.get(index).getDuration());
+
+            softAssert.assertThat(duration).isLessThan(10).as(voyages.get(index).getDuration() +
                     "is not less than 10");
-        }
+        });
```

Comment:
Do we really need index in this?
Why cant we use - response.getPayload().getVoyages().forEach()?